src/main/java/com/example/mate/model/Book.java

```diff
@@ -5,11 +5,23 @@
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
-@Data
+@Table(name = "books")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
do you really need both constructors?

Reply:
It is not necessary to use @AllArgsConstructor because 
I am using the MapStruct library. If I were using @Builder,
 then @AllArgsConstructor would be required, but in this case, it is not needed.